db/migrate/20220927115557_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :AuthorId, null: false, foreign_key: true
+      t.string :Title
+      t.text :Text
+      t.integer :CommentsCounter
+      t.integer :LikesCounter
+
+      t.timestamps
+    end
```

Comment:
Optional - I strongly suggest you change your key: values to lowercase 😄 😄

Reply:
Okay. I was just following the diagram and I thought they should appear the same